zone/worldserver.cpp

```diff
@@ -2114,6 +2114,24 @@ void WorldServer::HandleMessage(uint16 opcode, const EQ::Net::Packet &p)
 		}
 		break;
 	}
+	case ServerOP_ReloadNPCFactions:
+	{
+		if (zone && zone->IsLoaded()) {
+			zone->SendReloadMessage("NPC Factions");
+			zone->ReloadNPCFactions();
+		}
+
+		break;
+	}
+	case ServerOP_ReloadFactionAssociations:
+	{
+		if (zone && zone->IsLoaded()) {
+			zone->SendReloadMessage("Faction Associations");
+			zone->ReloadFactionAssociations();
+		}
+
+		break;
```

Comment:
We spoke about this but I think factions should all be reloaded together (all tables and types)